express-api/src/controllers/projects/projectsController.ts

```diff
@@ -92,14 +99,20 @@ export const deleteDisposalProject = async (req: Request, res: Response) => {
  * @returns {Response}      A 200 status with the new project.
  */
 export const addDisposalProject = async (req: Request, res: Response) => {
-  /**
-   * #swagger.tags = ['Projects']
-   * #swagger.description = 'Add a new disposal project.'
-   * #swagger.security = [{
-   *   "bearerAuth" : []
-   * }]
-   */
-  return stubResponse(res);
+  // Extract project data from request body
+  // Extract projectData and propertyIds from the request body
+  const {
+    project,
+    propertyIds,
+  }: { project: DeepPartial<Project>; propertyIds: ProjectPropertyIds } = req.body;
```

Comment:
Do we care about the length of the propertyIds arrays at all? Is a project with no properties allowed to be created?

Reply:
That's a good question, as this was possible in the old PIMS and a user could start creating a project save it and enter the properties at a later time.

Reply:
That being the case, I think I'm fine with keeping this as a valid option for now